### DISCUSSION TITLE
Team Meeting planning discussion 2022-02-08

### DISCUSSION THREAD

**Discussion by nalininsbs** [wpmarketingteam/Marcomms-Internal-Comms#10] [General]
Test discussion. To check how others can add under the WP organization

**Comment by yvettesonneveld** [comment]
Testing testing 123. This is a comment. 😄

**Comment by nalininsbs** [comment]
Updating this for discussions on the agenda for this week 2022-08-02:

Link to the template document https://docs.google.com/document/d/1TV6BBEa7E6dp_29bUyqiV2HJE5kdcEvL79Old_rAugs/edit#

**Comment by eidolonnight** [comment]
Updated formatting (some markdown no longer works in Slack) and added items for project updates. @nalininsbs do we have updates for any of the following?

- People of WordPress series
- Github transfer status
- WordCamp Europe 2022 Contributor Day
- WPDiversity

**Reply by nalininsbs** [reply]
Summary of reply to question in Slack
- brief update on People of WordPress
- Contributor Day/ events sub section - can incorporate WCEU, Yoast and if others are known or in the planning
- WPDiversity Group - new update I have been sent from @abhansnuk. This would also be useful for anyone wanting to assist on this in marketing. https://make.wordpress.org/community/2022/02/03/open-invitation-diverse-speaker-training-group-wpdiversity-zoom-call/ 

- GitHub (rather than transfer status): update

**Reply by nalininsbs** [reply]
@murillol to keep you connected.

**Comment by yvettesonneveld** [comment]
Maybe mention last Friday's contributor day in the Social media section, @eidolonnight?

**Comment by nalininsbs** [comment]
I suggest update on Yoast is done under a Contributor Day / events heading. We refer to WCEU, go to Yoast, and ask if any others are known about. From reading previous issues with identifying contributor events at camps and so on, there is no easy way to do this. This would be useful to ask people and for the question to be in the notes too rather than just a link to the Slack.

**Comment by yvettesonneveld** [comment]
Should we highlight this post in the meeting? https://make.wordpress.org/community/2022/02/08/discussion-diversity-in-wordpress-events/

**Comment by nalininsbs** [comment]
The option to close completed discussions is not showing. Only option to lock it. Will explore why this is.

**Reply by nalininsbs** [reply]
Tests:
- Converting the discussion to an issue to then close it on the allocated board - does not close the discussion ticket itself
